src/constants/commands.ts

```diff
@@ -27,6 +31,38 @@ export const GROUP_INVITE = {
     },
   ],
 };
+export const GRANT_AWS_ACCESS = {
+  name: "grant-aws-access",
+  description: "This command is to grant AWS access to the discord users.",
+  options: [
+    {
+      name: "user-name",
+      description: "User to be granted the AWS access",
+      type: 6, //user Id to be grant the access
+      required: true,
+    },
+    {
+      name: "aws-group-name",
+      description: "AWS group name",
+      type: 3,
+      required: true,
+      choices: [
+        {
+          name: "S3 read only access",
+          value: process.env.S3_READ_ONLY_ACCESS_AWS_GROUP_ID,
+        },
```

Comment:
We will never do this, I think

Reply:
Yeah these have to be replaced with actual values